src/tasks/dump.ts

```diff
@@ -353,13 +317,80 @@ export async function getDumpInstructions({
     transferToReceiver: await transferToReceiver,
   };
 }
+interface QuoteInput {
+  sellToken: Erc20Token;
+  buyToken: Erc20Token | NativeToken;
+  balance: BigNumber;
+  validTo: number;
+  maxFeePercent: number;
+  user: string;
+  api: Api;
+}
+
+// Returns null if the fee is not satisfying balance or maxFeePercent. May throw if an unexpected error occurs
+async function getQuote({
+  sellToken,
+  buyToken,
+  balance,
+  validTo,
+  maxFeePercent,
+  user,
+  api,
+}: QuoteInput): Promise<Quote | null> {
+  let quote;
+  try {
+    const quotedOrder = await api.getQuote({
+      sellToken: sellToken.address,
+      buyToken: isNativeToken(buyToken) ? BUY_ETH_ADDRESS : buyToken.address,
+      validTo,
+      appData: APP_DATA,
+      partiallyFillable: false,
+      from: user,
+      kind: OrderKind.SELL,
+      sellAmountBeforeFee: balance,
+    });
+    quote = {
+      sellAmount: BigNumber.from(quotedOrder.quote.sellAmount),
+      buyAmount: BigNumber.from(quotedOrder.quote.buyAmount),
+      feeAmount: BigNumber.from(quotedOrder.quote.feeAmount),
+    };
+  } catch (e) {
+    if ((e as CallError)?.apiError?.errorType === "SellAmountDoesNotCoverFee") {
```

Comment:
I imagine that `InsufficientLiquidity` errors could also be considered an "ignored" token.

Reply:
Also outside the scope of this PR, but here we could compare against `GetQuoteErrorType.SellAmountDoesNotCoverFee` instead of a fixed string.